src/pages-and-resources/discussions/data/api.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/prefer-default-export */
+export function getDiscussionApps() {
+  return Promise.resolve({
+    features: [
+      {
+        id: 'lti',
+        name: 'LTI Integration',
```

Comment:
are these needing i18n?

Reply:
This is going to be difficult to figure out.  Technically yes, since we'll want to be translating our feature list.  But because it's going to ultimately be a dynamic list fed from the backend... that gets a bit more complicated.  If we put the messages in the frontend and key into them using the IDs in this data structure, then it'll be a bit brittle, since the features list may change on the backend and we won't have a message for it.  But if we do it on the backend, then we're maintaining two separate i18n mechanisms (the django one and this one) which may not be desirable long term.

I'm going to guess that we'll do the former and build the messages in the frontend based on feature IDs and just let it be a bit brittle for the foreseeable future.  In either case, I'm not going to bother until we have the real data shape from the backend.

Reply:
OK. I think it's possible we *do* want an i18n ticket attached to some future milestone, so we can avoid thinking about it through these first few.